tests/shell/Makefile

```diff
@@ -6,13 +6,17 @@ USEMODULE += shell
 USEMODULE += shell_commands
 USEMODULE += ps
 
+# for z1, socat doesn't work (unknown reason)
+ifeq (z1, $(BOARD))
+  RIOT_TERMINAL ?= pyterm
+endif
+
```

Comment:
This doesn't seem to be a test-specific issue. Maybe add this to `tests/Makefile.tests_common` or somewhere more global?

Reply:
I just cross-checked tests/fmt_print with socat, that works. I honestly don't know what's going on. I'm assuming that socat's buffering, in combination with the FTDI and no-flow-control, sends data too fast. why that shows in the shell test, I don't understand.

Reply:
btw, this is cut out into https://github.com/RIOT-OS/RIOT/pull/14658